data/chinaso

```diff
@@ -0,0 +1,2 @@
+chinaso.com
```

Comment:
advertisement detected in `http://adq.chinaso.com/ad.min.js`.
consider mark `adq.chinaso.com` as ads.